packages/medusa/src/api-v2/store/payments/payment-providers/route.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  AuthenticatedMedusaRequest,
+  MedusaResponse,
+} from "../../../../types/routing"
+import {
+  ContainerRegistrationKeys,
+  remoteQueryObjectFromString,
+} from "@medusajs/utils"
+import { StoreGetPaymentProvidersParamsType } from "../validators"
+
+// TODO: The provider fetching need to be context-based, similar to shipping methods.
```

Comment:
Payment providers are configured per Region, so we could consider moving this endpoint:
```
GET /store/regions/:region_id/payment-providers
```

Alternatively, we would need to pass the `region_id` ID here. There's no other context to take into account.